cms/static/js/views/modals/base_modal.js

```diff
@@ -87,6 +87,9 @@ define(["jquery", "underscore", "gettext", "js/views/baseview"],
                 this.render();
                 this.resize();
                 $(window).resize(_.bind(this.resize, this));
+
+                // after showing and resizing, send focus
+                this.$el.find('.modal-window').focus();
```

Comment:
The class ".modal-window" is used 2 times in this file. Please promote it to a class variable to make the code easier to maintain.